src/atlas/meshgenerator/detail/CubedSphereMeshGenerator.cc

```diff
@@ -42,298 +45,348 @@ namespace meshgenerator {
 
 // -------------------------------------------------------------------------------------------------
 
-CubedSphereMeshGenerator::CubedSphereMeshGenerator( const eckit::Parametrisation& p ) {}
+CubedSphereMeshGenerator::CubedSphereMeshGenerator(const eckit::Parametrisation& p) {}
```

Comment:
I think that the local coding style is to have a space between brackets and things within them.

Reply:
I agree. That said, I'd have to redo almost every other line in my code. I think Willem has a script that correctly formats this.